web/ui/react-app/src/Panel.tsx

```diff
@@ -153,14 +154,29 @@ class Panel extends Component<PanelProps, PanelState> {
         throw new Error(json.error || 'invalid response JSON');
       }
 
+      let totalTimeSeries = 0;
+      if (json.data) {
+        const { resultType, result} = json.data;
+        if (resultType === "scalar") {
+          totalTimeSeries = 1;
+        } else if(result && result.length > 0) {
```

Comment:
Whitespace consistency nit:

```suggestion
        } else if (result && result.length > 0) {
```